src/common/definitions.js

```diff
@@ -126,6 +126,13 @@ const ssn = {
   pattern: '^[0-9]{9}$'
 };
 
+// Historically a veteran's service number has been between 5 and 8 digits, 
+// with the possibility of an alphabetical prefix of up to two capital letters.
+const veteranServiceNumber = {
```

Comment:
I elected to add this to the common definitions because it seems there's a possibility it may be used elsewhere. After speaking with folks at the DSVA, and doing some research, this pattern was chosen because historically, service numbers can have between 5 and 8 numbers, prefixed by up to two capital letters.